asreview/data/base.py

```diff
@@ -498,6 +498,9 @@ def duplicated(self, pid='doi'):
             s_dups_pid = ((s_pid.duplicated()) & (s_pid.notnull()))
         else:
             s_dups_pid = None
+           
+        if pid == 'doi':
+            s_pid = s_pid.str.replace("^https?://(www\.)?doi\.org/", "", regex=True)
```

Comment:
Very nice addition. For extra protection, I'd rather move this to
```python
        if pid in self.df.columns:
            # in case of strings, strip whitespaces and replace empty strings with None
            if is_string_dtype(self.df[pid]):
                s_pid = self.df[pid].str.strip().replace("", None)
                if pid == 'doi':
                    s_pid = s_pid.str.replace("^https?://(www\.)?doi\.org/", "", regex=True)
            else:
                s_pid = self.df[pid]
```

This prevents an error if the column "doi" contains something else than then you would expect.

Reply:
Replacing `"^https?://(www\.)?doi\.org/"` by `r"^https?://(www\.)?doi\.org/"` resolves on of the lint errors.